common/src/main/java/tachyon/conf/TachyonConf.java

```diff
@@ -42,14 +42,28 @@
 /**
  * <p>
  * Configuration of Tachyon. It sets various Tachyon parameters as key-value pairs. This class
- * contains all the runtime configuration properties. The default properties is stored in file
- * <code>tachyon-default.properties</code> and users can override these default properties by
- * modifying file <code>tachyon-site.properties</code>.
- * </p>
+ * contains all the runtime configuration properties.
+ *
+ * <p>
+ * Here is the order for this class to decide where to load the properties:
```

Comment:
Perhaps `Properties are loading in the following order, with decreasing priority`?